egs/wham/DynamicMixing/local/augmented_wham.py

```diff
@@ -0,0 +1,278 @@
+import os
```

Comment:
Let's keep it here for now and see if we move it to src later.

Reply:
Yes that was the idea, basically the class right now can do data-augmented wham alongside with plain wham and for orig_percentage==1 is just plain wham